app/src/components/VoteChoice.vue

```diff
@@ -16,13 +17,24 @@ export default {
     },
     methods: {
         voteUp: function () {
-            alert(this.id)
-            this.score--;
```

Comment:
I actually made service functions for current voting logic, but can merge later :D